test/factories_test.cc

```diff
@@ -34,20 +35,30 @@ TEST_F(FactoriesTest, CreateBenchmarkClient) {
   Envoy::Upstream::ClusterManagerPtr cluster_manager;
   EXPECT_CALL(options_, connections()).Times(1);
   EXPECT_CALL(options_, h2()).Times(1);
-  EXPECT_CALL(options_, requestMethod()).Times(1);
-  EXPECT_CALL(options_, requestBodySize()).Times(1);
   EXPECT_CALL(options_, maxPendingRequests()).Times(1);
   EXPECT_CALL(options_, maxActiveRequests()).Times(1);
   EXPECT_CALL(options_, maxRequestsPerConnection()).Times(1);
   auto cmd = std::make_unique<nighthawk::client::CommandLineOptions>();
+  EXPECT_CALL(options_, toCommandLineOptions()).Times(1).WillOnce(Return(ByMove(std::move(cmd))));
+  StaticHeaderSourceImpl header_generator(
+      Envoy::Http::HeaderMapPtr{new Envoy::Http::TestHeaderMapImpl{}});
```

Comment:
std::make_shared?

Reply:
Addressed in b51e654

Reply:
That commit doesn't effect this particular line.